test/Validate.CFG.cpp

```diff
@@ -707,9 +707,178 @@ TEST_F(ValidateCFG, DISABLED_BackEdgeBlockDoesntPostDominateContinueTargetBad) {
 
   CompileSuccessfully(str);
   ASSERT_EQ(SPV_ERROR_INVALID_CFG, ValidateInstructions());
+  EXPECT_THAT(getDiagnosticString(),
+              MatchesRegex("The continue construct with the continue target "
+                           ".\\[loop2\\] does not dominate the back-edge block "
```

Comment:
This isn't what the test name says it's going to test.
This test is failing on a dominance test, but the test name says it's going to check post-dominance.